test/e2e/traceflow_test.go

```diff
@@ -598,48 +596,6 @@ func testTraceflowIntraNode(t *testing.T, data *TestData) {
 			expectedPhase:   v1alpha1.Failed,
 			expectedReasons: []string{fmt.Sprintf("Node: %s, error: failed to get the destination Pod: pods \"%s\" not found", node1, "non-existing-pod")},
 		},
-		{
-			name:      "nonExistingSrcPodIPv4",
-			ipVersion: 4,
-			tf: &v1alpha1.Traceflow{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: randName(fmt.Sprintf("%s-%s-to-%s-%s-", data.testNamespace, "non-existing-pod", data.testNamespace, node1Pods[1])),
-				},
-				Spec: v1alpha1.TraceflowSpec{
-					Source: v1alpha1.Source{
-						Namespace: data.testNamespace,
-						Pod:       "non-existing-pod",
-					},
-					Destination: v1alpha1.Destination{
-						Namespace: data.testNamespace,
-						Pod:       node1Pods[1],
-					},
-				},
-			},
-			expectedPhase:   v1alpha1.Failed,
-			expectedReasons: []string{fmt.Sprintf("Invalid Traceflow request, err: %+v", fmt.Errorf("requested source Pod %s not found", k8s.NamespacedName(data.testNamespace, "non-existing-pod")))},
-		},
-		{
-			name:      "hostNetworkSrcPodIPv4",
-			ipVersion: 4,
-			tf: &v1alpha1.Traceflow{
-				ObjectMeta: metav1.ObjectMeta{
-					Name: randName(fmt.Sprintf("%s-%s-to-%s-%s-", antreaNamespace, agentPod, data.testNamespace, node1Pods[1])),
-				},
-				Spec: v1alpha1.TraceflowSpec{
-					Source: v1alpha1.Source{
-						Namespace: antreaNamespace,
-						Pod:       agentPod,
-					},
-					Destination: v1alpha1.Destination{
-						Namespace: data.testNamespace,
-						Pod:       node1Pods[1],
-					},
-				},
-			},
-			expectedPhase:   v1alpha1.Failed,
-			expectedReasons: []string{fmt.Sprintf("Invalid Traceflow request, err: %+v", fmt.Errorf("using hostNetwork Pod as source in non-live-traffic Traceflow is not supported"))},
-		},
```

Comment:
Could you keep these cases as it can verify the result of invalid request? UT cannot detect the issue on apiserver side.